lib/logitech_receiver/base.py

```diff
@@ -189,6 +198,23 @@ def read(handle, timeout=DEFAULT_TIMEOUT):
 		return reply[1:]
 
 
+# sanity checks on  message report id and size
+def check_message(data) :
+	assert isinstance(data, bytes), (repr(data), type(data))
+	report_id = ord(data[:1])
+	if report_id in report_lengths:  # is this an HID++ or DJ message?
+		if report_lengths.get(report_id) == len(data):
+			return True
+		else:
+			_log.warn("unexpected message size: report_id %02X message %s" % (report_id, _strhex(data)))
+			return False
+	else:
+# don't even bother to log other messages
+#		if _log.isEnabledFor(_DEBUG):
+#			_log.debug("ignoring message with unknown report id: %02X message %s", report_id, _strhex(data))
+		return False
```

Comment:
Remove the logging code, remove the else and only return false one time at the end.

Reply:
Done

Reply:
I force-pushed the change so as not to create another commit.